### PR TITLE
Context switcher 1

### PR DESCRIPTION
**Pull request by epwinchell**
Jira: https://issues.redhat.com/browse/RHCLOUD-12316

Screen Shot 2021-03-03 at 12 22 26 PM
Screen Shot 2021-03-03 at 12 22 35 PM

### CODE REVIEW

**Review by ryelo — APPROVED** [review]

**Comment by ryelo** [comment]
Eventually, we'll want to pass the user object into the context switcher too, similar to how we do it on the feedback button. I don't know if you want to go ahead and pass that prop in and actually fill in the account number, but it's there if you need it.

**Comment by epwinchell** [comment]
@ryelo Account number added

**Review by ryelo — APPROVED** [review]

File: src/js/App/Header/ContextSwitcher.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Dropdown, DropdownToggle } from '@patternfly/react-core';
+
+import './ContextSwitcher.scss';
+// import useGlobalNav from '../../utils/useGlobalNav';
```
Comment (Hyperkid123) [inline src/js/App/Header/ContextSwitcher.js:6]: Can you delete the comment, please?

File: src/js/App/Header/ContextSwitcher.scss [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+        }
+        0% {
+          background-color: var(--pf-global--active-color--300);
+        }
+      }
+    }
+  }
+}
+
+.mm-test {
```
Comment (Hyperkid123) [inline src/js/App/Header/ContextSwitcher.scss:51]: we might want a different class name

Comment (epwinchell) [reply src/js/App/Header/ContextSwitcher.scss:51]: @Hyperkid123 It was in your honor, but fine...

**Pull request merged**